tests/utils.py

```diff
@@ -0,0 +1,91 @@
+# This file is part of meas_extensions_scarlet.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+import scipy.signal
+
+from lsst.geom import Box2I, Point2I, Extent2I
+from lsst.afw.geom import Span, SpanSet
+from lsst.afw.detection import Footprint, GaussianPsf
+
+
+def numpyToStack(images, center, offset):
+    """Convert numpy and python objects to stack objects
+    """
+    cy, cx = center
+    bands, height, width = images.shape
+    x0, y0 = offset
+    bbox = Box2I(Point2I(x0, y0), Extent2I(width, height))
+    spans = [Span(y, x0, x0+width-1) for y in range(y0, y0+height)]
```

Comment:
You can just make a spanset from a box here

Reply:
I didn't realize that you could initialize with `SpanSet(bbox)` until I actually looked at the code. It might be a good idea to add this initialization to the docs.

Reply:
Created DM-20776